Formula/bazel.rb

```diff
@@ -16,11 +16,14 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Force Bazel to put all temporary file in the homebrew cache
+    ENV["BAZEL_WRKDIR"] = "#{buildpath}"
```

Comment:
I'm not entirely sure how the contents of `BAZEL_WRKDIR` are structured, but it could make sense to confine them all to a subdirectory, e.g. `#{buildpath}/work`, as `#{buildpath}` is also the location where the source tarball gets extracted to.